docs/decisions/arch-diagrams/mobile-create-order.mmd

```diff
@@ -0,0 +1,73 @@
+sequenceDiagram
+    participant User
+    participant MobileApp
+    participant CTBackend as CT Backend
+    participant IAPValidator as IAP Validator (Android/iOS)
+    participant MobileStore as Mobile store (Android/iOS)
+    participant CoCo as Commercetools (CoCo)
+    participant EventSystem
+
+    User->>MobileApp: Starts in-app purchase
+    MobileApp->>CTBackend: Sends purchase info (price, course, platform, token)
+
+    CTBackend->>CoCo: Check if user already has a cart
```

Comment:
Before this step, it checks for the CT customer existence. If it does not exist, it creates one.